### PR TITLE
DOC-9004 Create a physical cluster replication metrics dashboard page when PCR goes GA

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-9004

(1) Added ui-physical-cluster-replication-dashboard.md, moved content from DB Console section of physical-cluster-replication-monitoring.md.
(2) In physical-cluster-replication-monitoring.md, ui-overview.md and self-hosted-deployments.json, added links to ui-physical-cluster-replication-dashboard.md.
(3) In ui-replication-dashboard.md, added callout regarding ui-physical-cluster-replication-dashboard.md.

### CODE REVIEW

**Review by kathancox — APPROVED** [review]
This looks good to me Florence. Could you also add the preview label to the top of the new dashboard page?

File: src/current/v24.1/ui-physical-cluster-replication-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+docs_area: reference.db_console
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+The **Physical Cluster Replication** dashboard in the DB Console lets you monitor the [physical cluster replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) streams between a primary and standby cluster.
+
+To view this dashboard, [access the DB Console]({% link {{ page.version.version }}/ui-overview.md %}#db-console-access) for your standby cluster, click **Metrics** on the left-hand navigation bar, and then select **Physical Cluster Replication** from the **Dashboard** dropdown.
```
Comment (taroface) [inline src/current/v24.1/ui-physical-cluster-replication-dashboard.md:14]:
```suggestion
To view this dashboard, [access the DB Console]({% link {{ page.version.version }}/ui-overview.md %}#db-console-access) for your standby cluster, click **Metrics** on the left-hand navigation bar, and select **Physical Cluster Replication** from the **Dashboard** dropdown.
```

File: src/current/v24.1/ui-physical-cluster-replication-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+{% include {{ page.version.version }}/ui/ui-metrics-navigation.md %}
+
+The **Physical Cluster Replication** dashboard displays the following time-series graphs:
+
+## Logical bytes
+
+<img src="{{ 'images/v24.1/ui-logical-bytes.png' | relative_url }}" alt="DB Console Logical Bytes graph showing results over the past hour" style="border:1px solid #eee;max-width:100%" />
+
+The **Logical Bytes** graph shows you the throughput of the replicated bytes. The graph displays the rate at which the logical bytes (sum of keys + values) are ingested by all replication jobs.
```
Comment (taroface) [inline src/current/v24.1/ui-physical-cluster-replication-dashboard.md:30]:
```suggestion
The **Logical Bytes** graph displays the throughput of the replicated bytes. The graph displays the rate at which the logical bytes (sum of keys and values) are ingested by all replication jobs.
```

File: src/current/v24.1/ui-physical-cluster-replication-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+{{site.data.alerts.callout_info}}
+When you [start a replication stream]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}#step-4-start-replication), the **Logical Bytes** graph will record a spike of throughput as the initial scan completes. {% comment %}link to technical details here{% endcomment %}
+{{site.data.alerts.end}}
+
+## SST bytes
+
+<img src="{{ 'images/v24.1/ui-sst-bytes.png' | relative_url }}" alt="DB Console SST bytes graph showing results over the past hour" style="border:1px solid #eee;max-width:100%" />
+
+The **SST Bytes** graph shows you the rate at which all [SST]({% link {{ page.version.version }}/architecture/storage-layer.md %}#ssts) bytes are sent to the [KV layer]({% link {{ page.version.version }}/architecture/storage-layer.md %}) by physical cluster replication jobs.
```
Comment (taroface) [inline src/current/v24.1/ui-physical-cluster-replication-dashboard.md:45]:
```suggestion
The **SST Bytes** graph displays the rate at which all [SST]({% link {{ page.version.version }}/architecture/storage-layer.md %}#ssts) bytes are sent to the [KV layer]({% link {{ page.version.version }}/architecture/storage-layer.md %}) by physical cluster replication jobs.
```

**Review by taroface — APPROVED** [review]
LGTM. Some minor suggestions.

File: src/current/v24.1/ui-physical-cluster-replication-dashboard.md (hunk 1 above)
Comment (florence-crl) [reply src/current/v24.1/ui-physical-cluster-replication-dashboard.md]: updated

File: src/current/v24.1/ui-physical-cluster-replication-dashboard.md (hunk 2 above)
Comment (florence-crl) [reply src/current/v24.1/ui-physical-cluster-replication-dashboard.md]: updated

File: src/current/v24.1/ui-physical-cluster-replication-dashboard.md (hunk 3 above)
Comment (florence-crl) [reply src/current/v24.1/ui-physical-cluster-replication-dashboard.md]: updated

**Review by florence-crl — COMMENTED** [review]
TFTRs!

**Pull request merged**